docs/_static/custom.css

```diff
@@ -194,21 +208,128 @@ footer {
   h2 {
     font-size: 34px;
     line-height: 34px;
-    font-family: "Space Mono", sans-serif;
+    font-family: 'Barlow', sans-serif;
     letter-spacing: -0.05em;
   }
 
-  a {
-    color: #c2006b;
-    text-decoration: underline;
-    cursor: pointer;
-  }
+ 
+}
 
-  a:visited {
-    color: #c2006b;
-  }
 
-  a:hover {
-    color: rgb(0, 0, 0);
+
+/*AG LINKS */
+a {
+  color:rgb(93, 0, 255);
+  cursor: pointer;
+  
+}
+
+
+@media only screen and (min-device-width: 320px) and (max-device-width: 480px) {
+
+a {
+  color:rgb(93, 0, 255);
+  cursor: pointer;
+}
+
+}
+
+/*NEW AG ADDITIONS * EVEN NEWERE - TRYING TO MESS WITH THT SIDEBAR
```

Comment:
Some these comments are kind of distracting. Why all caps and alternative spellings?

It is very characteristic of you, but I am not it is necessary in this document.

Reply:
Yeah let's clean these out :heart: